Makefile

```diff
@@ -52,8 +52,8 @@ infra.up: ## Setup localstack for development
 
 testacc:
 	TF_ACC=1 \
-		MASSDRIVER_AWS_ENDPOINT=http://localhost:4566 \
```

Comment:
localstack can't be reached via `localhost` from within the dev container. We need to use the docker bridge network alias `localstack`.